CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+## JavaScript
+- Fixed a bug where if any injected dependencies to a JavaScript (e.g. `ux:Name`, `ux:Dependency`, `dep:`) have names that correspond to EcmaScript 5.1 reserved words, these dependencies will now be available with a dollar sign `$` prefix (e.g. `$undefined`) instead of causing a script compile error.
```

Comment:
Read my comment again. The motivation of this change is *just* as backwards incompatible as injecting $foo in addition to injecting foo.

The second part of your comment has nothing to do with what I'm talking about.